checks.d/http_check.py

```diff
@@ -288,8 +290,10 @@ def _check(self, instance):
             else:
                 expected_code = http_response_status_code
 
-            message = "Incorrect HTTP return code for url %s. Expected %s, got %s" % (
+            message = "Incorrect HTTP return code for url %s. Expected %s, got %s." % (
                 addr, expected_code, str(r.status_code))
+            if include_content:
+                message += '\nContent: {}'.format(r.content[:CONTENT_LENGTH])
 
             self.log.info(message)
```

Comment:
Shall we add it here too: https://github.com/DataDog/dd-agent/blob/master/checks.d/http_check.py#L315-L319 ?